test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -102,7 +92,7 @@ export default class GutenbergEditorComponent extends AbstractEditorComponent {
 	async getTitle() {
 		return await this.driver
 			.findElement( By.css( '.editor-post-title__input' ) )
-			.getAttribute( 'value' );
+			.getProperty( 'textContent' );
```

Comment:
We should be able to use the `getInputText` helper suggested above.

Reply:
BTW, does `getProperty` work with Selenium's WebDriver? 🤔 I can't seem find it in [the docs](https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/lib/webdriver_exports_WebElement.html).

Reply:
Yep, it works. I think we `getText()` would work too.